src/openrct2/windows/ride.c

```diff
@@ -4011,7 +4011,23 @@ static void window_ride_maintenance_paint(rct_window *w, rct_drawpixelinfo *dpi)
 	if (ride->lifecycle_flags & RIDE_LIFECYCLE_BROKEN_DOWN) {
 		switch (ride->mechanic_status) {
 		case RIDE_MECHANIC_STATUS_CALLING:
```

Comment:
If you check the build status at [Travis CI](https://travis-ci.org/OpenRCT2/OpenRCT2/builds/222249258), it seems like the reason the build is failing there is because of [this error](http://stackoverflow.com/questions/92396/why-cant-variables-be-declared-in-a-switch-statement). I'd suggest adding the whole `case RIDE_MECHANIC_STATUS_CALLING:` case inside a block, like link suggests.